sdk/storage/azure-storage-blob/azure/storage/blob/_container_client_helpers.py

```diff
@@ -155,7 +156,14 @@ def _generate_delete_blobs_options(
 
         query_parameters, header_parameters = _generate_delete_blobs_subrequest_options(client, **options)
 
-        req = HttpRequest(
+        if azurite_name:
+            req = HttpRequest(
+            "DELETE",
+            f"/{quote(azurite_name)}/{quote(container_name)}/{quote(str(blob_name), safe='/~')}{query_str}",
+            headers=header_parameters
+        )
+        else:
+            req = HttpRequest(
```

Comment:
Looking for feedback if we should leave it like this, or maybe do a `not` to make the requests that matters 99.99% of the time be the first one, or even bring out the
`f"/{quote(azurite_name)}/{quote(container_name)}/{quote(str(blob_name), safe='/~')}{query_str}",` 
so that we can conditionally set that on some variable, and then just have one `req=`

Reply:
Combine ifs, use `f"{("/" + quote(prepend)) if prepend else ""}..."`